custom/conf/app.example.ini

```diff
@@ -193,8 +193,8 @@ RUN_USER = ; git
 ;; Use `ssh-keygen` to parse public SSH keys. The value is passed to the shell. By default, Gitea does the parsing itself.
 ;SSH_KEYGEN_PATH =
 ;;
-;; Enable SSH Authorized Key Backup when rewriting all keys, default is true
-;SSH_AUTHORIZED_KEYS_BACKUP = true
+;; Enable SSH Authorized Key Backup when rewriting all keys, default is false
+;SSH_AUTHORIZED_KEYS_BACKUP = false
```

Comment:
Isn't that breaking?
Or did it previously display the wrong value?

Reply:
Set SSH_AUTHORIZED_KEYS_BACKUP to false #25412

Reply:
> Isn't that breaking? Or did it previously display the wrong value?

This is just a missed documentation from #25412